chapter02/spg/question1.pl

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use File::Spec;
+use Cwd;
+
+my $dir = Cwd::getcwd;

```

Comment:
`Cwd` exports this by default, so it doesn't need a fully qualified call (unless you dislike the ambiguity of functions imported to the current namespace, in which case :+1:).